packages/lib/formbricks.ts

```diff
@@ -0,0 +1,28 @@
+import { FormbricksAPI } from "@formbricks/api";
+
+enum Rating {
+  "Extremely unsatisfied" = 1,
+  "Unsatisfied" = 2,
+  "Satisfied" = 3,
+  "Extremely satisfied" = 4,
+}
+
+if (!process.env.NEXT_PUBLIC_FORMBRICKS) throw new Error("Missing NEXT_PUBLIC_FORMBRICKS env variable");
+const [apiHost, environmentId, surveyId] = process.env.NEXT_PUBLIC_FORMBRICKS.split("+");
+const api = new FormbricksAPI({
+  apiHost,
+  environmentId,
+});
+
+export const submitSurveyResponse = async (rating: string, comment: string) => {
```

Comment:
please rename to `submitFeedback` or similar to make clear that this is to submit the feedback from the feedback box

Reply:
renamed it to `submitFeedbackFormbricks` to maintain their pattern of `sendFeedbackEmail` for email